packages/dx-react-grid/docs/reference/grouping-panel.md

```diff
@@ -56,7 +57,7 @@ allowDragging | boolean | Specifies whether an end-user can change the grouping
 sorting | Array&lt;[Sorting](sorting-state.md#sorting)&gt; | The current sorting state.
 changeSortingDirection | ({ keepOther: boolean, cancel: boolean, columnName: string }) => void | Changes the specified column's sorting direction. Keeps the current sorting state if `keepOther` is set to true. Cancels sorting by the specified column if `cancel` is set to true.
 groupingPanelItems | Array&lt;[GroupingPanelItem](#grouping-panel-item)&gt; | The Grouping Panel items.
-groupByColumnText | string | The text displayed in the group panel if grid data is not grouped.
+getMessage | ([messageKey](#localization-messages): string) => string | Returns the text displayed in the group panel if grid data is not grouped.
```

Comment:
Other plugins' templates has the same required property. Why it is described only in the GroupingPanel reference?